src/content/docs/en/guides/astro-db.mdx

```diff
@@ -499,3 +499,21 @@ export default async function() {
 
 The value returned by `asDrizzleTable('Pets', Pets)` is equivalent to `import { Pets } from 'astro:db'`, but is available even when Astro’s type generation can’t run.
 You can use it in any integration code that needs to query or insert into the database.
+
+## Self-hosted production deployment
+
+If you deploy your site to a self-managed host such as a [virtual private server](https://en.wikipedia.org/wiki/Virtual_private_server), you can choose to use a database file instead of connecting to Astro Studio.
```

Comment:
```suggestion
If you deploy your site to a self-managed host such as a [virtual private server](https://en.wikipedia.org/wiki/Virtual_private_server), you can choose to use a database file instead of connecting to a database hosted at Astro Studio.
```